pkg/generators/column.go

```diff
@@ -32,3 +38,20 @@ func CreateIndexesForColumn(table *typedef.Table, maxIndexes int) []typedef.Inde
 	}
 	return indexes
 }
+
+func AddReferencesForIndexes(table *typedef.Table) {
+	wrongIndex := -1
+	for i, index := range table.Indexes {
+		for c, column := range table.Columns {
+			if index.ColumnName == column.Name {
+				table.Indexes[i].Column = table.Columns[c]
+				break
+			} else if len(table.Columns) == c {
+				wrongIndex = i
+			}
+		}
+	}
+	if wrongIndex != -1 {
+		panic(fmt.Sprintf("wrong column_name in index defenition:%+v", table.Indexes[wrongIndex]))
+	}
+}
```

Comment:
```suggestion
        indexLoop:
	for i, index := range table.Indexes {
		for c, column := range table.Columns {
			if index.ColumnName == column.Name {
				table.Indexes[i].Column = table.Columns[c]
				continue indexLoop
			}
		}
		return errors.Wrapf(ErrSchemaValidation, "wrong column_name in index defenition:%+v", table.Indexes[wrongIndex])
	}
}
```

Reply:
interesting 'continue' use. I don`t know about like possibility.
But i removed error processing from this function. I suggest think that indexes already verified.
All verification realize in the future, in a separate function, as part of the work on: https://github.com/scylladb/gemini/issues/358d